### DISCUSSION TITLE
Delete a sharegpt convo

### DISCUSSION THREAD

**Discussion by zephyrkeaton** [vercel/community#2499] [Help] [answered]
### Summary

Hi
Yesterday I shared a convo from ChatGPT,  and I got a link like[https://sharegpt.com/c/*****], now I need to delete this convo, could you guys tell me how to delete it please, thank you.

**Comment by amyegan** [comment, marked as answer]
Hi @zephyrkeaton! Please reach out to the ShareGPT owners in https://github.com/domeccleston/sharegpt/discussions. They're the folks best able to help you with data removal 🙏

**Reply by zephyrkeaton** [reply]
Thankyou I'll try it!
